Networking/Networking/Mapper/ProductVariationsBulkCreateMapper.swift

```diff
@@ -34,21 +38,17 @@ struct ProductVariationsBulkCreateMapper: Mapper {
 /// This entity allows us to do parse all the things with JSONDecoder.
 ///
 private struct ProductVariationsEnvelope: Decodable {
```

Comment:
nit: maybe `ProductVariationsContainerEnvelope` would make this more obvious that it's `ProductVariationsContainer` with a data envelope, instead of a different struct

Reply:
Done in 35dc049a6ac7f9ca4584a99c1e59cef688afd671.